server/index.js

```diff
@@ -66,6 +48,16 @@ parsers.use(bodyParser.json());
 parsers.use(bodyParser.urlencoded({extended: false}));
 parsers.use(cookieParser());
 
+database.selectApplicationConfig(process.env.XIVELY_ACCOUNT_ID).then(function(data) {
+  console.log('Provisioning data loaded.');
+  var Config = require('./config');
+  config.raw = new Config(data[0]);
+  // Allow config to be referenced by name inside of views.
+  // This creates a new object that has everything in config plus an
+  // additional property "config" that is a circular reference
+  config.view = _.merge({}, config.raw, {config: config.raw});
```

Comment:
I remember shrugging at this line of code before, seems like an oddity to me but I'm not sure. I think it is used in the views, but I'm not sure how. For another day and another PR, any thoughts on this setup?